charts/metallb/templates/speaker.yaml

```diff
@@ -341,6 +352,16 @@ spec:
         resources:
           {{- toYaml . | nindent 12 }}
         {{- end }}
+        livenessProbe:
+          exec:
+            command: ["/etc/frr_liveness/liveness.sh"]
+          initialDelaySeconds: 5
+          periodSeconds: 5
```

Comment:
Shouldn't this be: 
            periodSeconds: 5
            failureThreshold: 3
like in different places in the code?

Reply:
yeah that's the old version, good catch

Reply:
fixed